community-letter-sprint-4-robusta-beans.md

```diff
@@ -0,0 +1,39 @@
+Hi folks!
+
+In the three coming weeks, me, Duc TRAN, René CORDIER, and Dat PHAM will be working on two cool features: "Event Dead Letter" and "Deleted Messages Vault"
```

Comment:
reviewing is not working? :)

Reply:
Good question.

I did wanted to focus on individual contributors (common stuff in Apache) and I considered your main focus for the coming weeks is elsewhere...

I can very well add other names here if you want.

Reply:
I approved, it's not a big matter for me. In fact naming contributors before they work on feature sound strange to me, because it does not invite other people to help contributing on this.

Reply:
True but we are going to work on this, enforcing 1. quality 2. tight dead line. 

So even if I truely want as many contributors to come and help us this do not really seem doable.